### PR TITLE
MC-6354: updated instances api docs

### PR DESCRIPTION
**Pull request by anuragkalra**

### CODE REVIEW

**Review by UdaySahni — APPROVED** [review]

**Review by Havarem — APPROVED** [review]
💯

File: source/includes/openstack/_instances.md [hunk 1]
```diff
@@ -135,6 +135,10 @@ Required | &nbsp;
 `networkId`<br/>*UUID* | The network in which the instance will be created. If you don't have a network, it can be created through the create network API.
 `securityGroupNames`<br/>*Array[string]* | The list of [security groups](#security-groups) to be associated to the instance
 
+Optional | &nbsp;
+------ | -----------
+`userData`<br/>*string* | Data that the user can specify when they launch an instance
```
Comment (prollynomial) [inline source/includes/openstack/_instances.md]: This isn't very descriptive - you should include something about cloud-init to make it more informative.

**Review by prollynomial — CHANGES_REQUESTED** [review]

File: source/includes/openstack/_instances.md (hunk 1 above)
Comment (anuragkalra) [inline source/includes/openstack/_instances.md]: On it

Comment (anuragkalra) [inline source/includes/openstack/_instances.md]: Fixed

File: source/includes/openstack/_instances.md [hunk 2]
```diff
@@ -135,6 +135,10 @@ Required | &nbsp;
 `networkId`<br/>*UUID* | The network in which the instance will be created. If you don't have a network, it can be created through the create network API.
 `securityGroupNames`<br/>*Array[string]* | The list of [security groups](#security-groups) to be associated to the instance
 
+Optional | &nbsp;
+------ | -----------
+`userData`<br/>*string* | Data that the user can specify when they launch an instance. User data holds a file that cloud-aware applications in the guest instance can access. One application that uses user data is the cloud-init system, which is available on various Linux systems.
```
Comment (prollynomial) [inline source/includes/openstack/_instances.md]: Is this your own writing?

Comment (anuragkalra) [inline source/includes/openstack/_instances.md]: I looked at https://docs.openstack.org/user-guide/cli-provide-user-data-to-instances.html to write it, it uses some of their terminology. Should I cite them or change it in some way? What is your recommendation

Comment (anuragkalra) [inline source/includes/openstack/_instances.md]: How is: "Data that the user can specify when they launch an instance. User data can be used in the cloud-init system as a script that is run during the initial launch. The user data field can be used to configure SSH keys, user settings, or run a script."

Comment (prollynomial) [inline source/includes/openstack/_instances.md]: That last version looks good to me.

Comment (anuragkalra) [inline source/includes/openstack/_instances.md]: cool I'm on it

Comment (anuragkalra) [inline source/includes/openstack/_instances.md]: Done

**Review by prollynomial — APPROVED** [review]

**Pull request merged**